### PR TITLE
Add StyledDropdown

### PR DESCRIPTION
**Pull request by vforge**
- Add styled version of Dropdown and List
- Bump some dev deps
- Re-snap

### CODE REVIEW

File: source/systems/BannerNotifications/BannerNotificationsComponent.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const mapStateToProps = state => ({
     messages: state.BannerNotificationsStore.getNotifications().toJS(),
 });
 
-const mapDispatchToProps = dispatch => ({
+const mapDispatchToProps = /* istanbul ignore next */ dispatch => ({
```
Comment (JoshuaRogan) [inline source/systems/BannerNotifications/BannerNotificationsComponent.js]: Why did you have to ignore this test?

Comment (vforge) [reply source/systems/BannerNotifications/BannerNotificationsComponent.js]: Apparently the disable that's already inside the function wasn't enough - the `onClose` function was reported as untested. I updated jest and some other libs so that might be the issue.

**Comment by JoshuaRogan** [comment]
Pretty neat to see how easily a css/scss component converts over to styled component. It makes more sense to me in styled-components nowadays.

**Comment by vforge** [comment]
@JoshuaRogan It's really hard to follow some complex nesting in SCSS, but I opted in favor of keeping the styles as close to the source material as possible.
